pinot-query-planner/src/test/java/org/apache/calcite/rel/rules/PinotSortExchangeCopyRuleTest.java

```diff
@@ -81,7 +81,8 @@ public void tearDown()
   @Test
   public void shouldMatchLimitNoOffsetNoSort() {
     // Given:
-    SortExchange exchange = LogicalSortExchange.create(_input, RelDistributions.SINGLETON, RelCollations.EMPTY);
+    SortExchange exchange = PinotLogicalSortExchange.create(_input, RelDistributions.SINGLETON, RelCollations.EMPTY,
+        false, true);
```

Comment:
So testing this is a bit tricky since on the Planner side I don't set any combination except `(false, true)` for now. These tests don't actually create nodes where we fail the creation of the nodes. I was planning to add tests with the hints once we add hint support to ensure that if the hint is set we fail since we don't support ordering on send.

Okay to take this up when hint support is added?